address-space-controller/src/main/java/io/enmasse/controller/ControllerChain.java

```diff
@@ -114,18 +114,27 @@ public void onUpdate(List<AddressSpace> resources) throws Exception {
         metrics.reportMetric(new Metric("version", new MetricValue(0, now, new MetricLabel("name", "address-space-controller"), new MetricLabel("version", version))));
         metrics.reportMetric(new Metric("health", new MetricValue(0, now, new MetricLabel("status", "ok"), new MetricLabel("summary", "address-space-controller is healthy"))));
 
-        List<MetricValue> healthValues = new ArrayList<>();
+        List<MetricValue> readyValues = new ArrayList<>();
         for (AddressSpace addressSpace : resources) {
-            List<MetricLabel> healthLabels = new ArrayList<>();
-            healthLabels.add(new MetricLabel("name", addressSpace.getName()));
-            healthLabels.add(new MetricLabel("messages", String.join(",", addressSpace.getStatus().getMessages())));
-            healthValues.add(new MetricValue(addressSpace.getStatus().isReady() ? 1 : 0, now, healthLabels));
+            readyValues.add(new MetricValue(addressSpace.getStatus().isReady() ? 1 : 0, now, new MetricLabel("name", addressSpace.getName())));
         }
+
+
+        List<MetricValue> pendingValues = new ArrayList<>();
+        for (AddressSpace addressSpace : resources) {
+            pendingValues.add(new MetricValue(addressSpace.getStatus().isReady() ? 0 : 1, now, new MetricLabel("name", addressSpace.getName())));
+        }
+        
         metrics.reportMetric(new Metric(
                 "address_spaces_ready",
                 "Address Spaces Ready",
                 MetricType.gauge,
-                healthValues));
+                readyValues));
+        metrics.reportMetric(new Metric(
+                    "address_spaces_pending",
+                    "Address Spaces Pending",
```

Comment:
I'm not a native english writer/speaker, but is pending the right word to use? To me, pending means it has been created and is waiting to be ready, whereas in this case it can move to pending even after it has been running successfully for a while.